Kickstarter-iOS/Views/Controllers/RootTabBarViewController.swift

```diff
@@ -108,49 +108,47 @@ public final class RootTabBarViewController: UITabBarController {
 
   // swiftlint:disable:next cyclomatic_complexity
   // swiftlint:disable:next function_body_length
-  private func setTabBarItemStyles(withData data: TabBarItemsData) {
+  fileprivate func setTabBarItemStyles(withData data: TabBarItemsData) {
     data.items.forEach { item in
       switch item {
       case let .home(index):
-        tabBarItem(atIndex: index) ?|> homeTabBarItemStyle(isMember: data.isMember)
+        _ = tabBarItem(atIndex: index) ?|> homeTabBarItemStyle(isMember: data.isMember)
       case let .activity(index):
-        tabBarItem(atIndex: index) ?|> activityTabBarItemStyle(isMember: data.isMember)
+        _ = tabBarItem(atIndex: index) ?|> activityTabBarItemStyle(isMember: data.isMember)
       case let .search(index):
-        tabBarItem(atIndex: index) ?|> searchTabBarItemStyle
+        _ = tabBarItem(atIndex: index) ?|> searchTabBarItemStyle
       case let .dashboard(index):
-        tabBarItem(atIndex: index) ?|> dashboardTabBarItemStyle
+        _ = tabBarItem(atIndex: index) ?|> dashboardTabBarItemStyle
       case let .profile(avatarUrl, index):
-        tabBarItem(atIndex: index) ?|> profileTabBarItemStyle(isLoggedIn: data.isLoggedIn,
-          isMember: data.isMember)
+        _ = tabBarItem(atIndex: index)
+          ?|> profileTabBarItemStyle(isLoggedIn: data.isLoggedIn, isMember: data.isMember)
 
         guard
           data.isLoggedIn == true,
           let avatarUrl = avatarUrl,
-          let dir = NSSearchPathForDirectoriesInDomains(.CachesDirectory, .UserDomainMask, true).first,
-          let hash = avatarUrl.absoluteString?.hashValue
-          else {
-            return
-        }
+          let dir = NSSearchPathForDirectoriesInDomains(.cachesDirectory, .userDomainMask, true).first
+          else { return }
 
+        let hash = avatarUrl.absoluteString.hashValue
```

Comment:
so i think this hash should be inside the `guard` clause above, otherwise looks good!